### PR TITLE
feat(groupbehavior): modify message grouping behavior AP-107

### PR DESCRIPTION
**Pull request by nickjjang**
AP-107

### CODE REVIEW

**Review by RetroPronghorn — APPROVED** [review]

File: utilities/Messaging.ts [hunk 1]
```diff
@@ -63,6 +63,20 @@ export function groupMessages(
       ? dayjs(currentMessage.at).isSame(prevMessage.at, 'day')
       : false
 
+    let isSameGroup
+    if (!prevMessage) {
+      isSameGroup = false
+    } else {
+      const currentAt = dayjs(currentMessage.at)
+      const prevAt = dayjs(prevMessage.at)
+      if (!dayjs().isSame(currentAt, 'day')) {
```
Comment (Jekrimo) [inline utilities/Messaging.ts:72]: Can we Enum the strings here for day, minutes, and unknown below?

Comment (nickjjang) [reply utilities/Messaging.ts:72]: I just follow the former coders, and dayjs was not made by me so I dont need to define enum for it.
I've checked that in dayjs they define variable like 
 `export type UnitTypeLong = 'millisecond' | 'second' | 'minute' | 'hour' | 'day' | 'month' | 'year' | 'date'`
so I followed their definition
@Jekrimo would you like to bring this issue in the meeting?

**Review by Jekrimo — APPROVED** [review]

**Comment by nickjjang** [comment]
@stavares843 will you merge?

**Comment by stavares843** [comment]
taking a look!

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**